tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -160,30 +160,22 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 
     fluid.tests.tooltipModuleSource = function (pager) {
         var sequence = [];
+        var linksList = pager.dataModel.map(function(data) {
+            return {text: data.animal};
+        });
```

Comment:
Could you use the `fluid.transform` utility for this. http://docs.fluidproject.org/infusion/development/CoreAPI.html#fluid-transform-source-fn1-fn2-